ui/raidboss/data/05-shb/raid/e9s.js

```diff
@@ -1,8 +1,384 @@
+          ko: '레이저 피하기',
+        },
+      },
+    },
+    {
+      id: 'E9S Obscure Woods',
+      netRegex: NetRegexes.startsUsing({ id: '55EE', source: 'Cloud Of Darkness', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 5,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E9S Flood Of Obscurity',
+      netRegex: NetRegexes.startsUsing({ id: '5907', source: 'Cloud Of Darkness', capture: false }),
+      delaySeconds: 3,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Place Bramble',
+        },
+      },
+    },
+    {
+      id: 'E9S Waste Away',
+      netRegex: NetRegexes.startsUsing({ id: '5617', source: 'Cloud Of Darkness', capture: false }),
+      response: Responses.knockback(),
+    },
+    {
+      id: 'E9S Rejuvenating Balm',
+      netRegex: NetRegexes.startsUsing({ id: '5618', source: 'Cloud Of Darkness', capture: false }),
+      durationSeconds: 5,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Away From Tethered Walls',
+        },
+      },
+    },
+    {
+      id: 'E9S Stygian Break Tether',
+      netRegex: NetRegexes.headMarker({ id: '00D4' }),
+      condition: Conditions.targetIsYou(),
+      delaySeconds: 3,
+      response: Responses.breakChains(),
+    },
+    {
+      id: 'E9S Anti-Air Phaser Unlimited Out',
+      netRegex: NetRegexes.startsUsing({ id: '561[23]', source: 'Cloud Of Darkness', capture: false }),
+      delaySeconds: 1,
+      durationSeconds: 6,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Go Tethered Wall Then Out',
+        },
+      },
+    },
+    {
+      id: 'E9S Anti-Air Phaser Unlimited Soaks',
+      netRegex: NetRegexes.startsUsing({ id: '561[23]', source: 'Cloud Of Darkness', capture: false }),
+      delaySeconds: 7,
+      alarmText: function(data, _, output) {
+        if (data.role === 'tank')
+          return output.stayOut();
+      },
+      alertText: function(data, _, output) {
+        if (data.role !== 'tank')
+          return output.healerStacks();
+      },
+      outputStrings: {
+        stayOut: {
+          en: 'Stay Out',
+          de: 'Draußen stehen',
+          fr: 'Restez éloigné',
+          ja: 'ライトニングを外に安置',
+          cn: '外侧放雷',
+          ko: '바깥에 있기',
+        },
+        healerStacks: {
+          en: 'Healer Stacks',
+          de: 'Bei den Heilern sammeln',
+          fr: 'Packages sur les heals',
+          ja: 'ヒーラーに集合',
+          cn: '治疗集合',
+          ko: '힐러 모이기',
+        },
+      },
+    },
+    {
+      id: 'E9S Anti-Air Phaser Unlimited Sides',
+      netRegex: NetRegexes.startsUsing({ id: '561[23]', source: 'Cloud Of Darkness', capture: false }),
+      delaySeconds: 11,
+      response: Responses.goSides(),
+    },
+    {
+      id: 'E9S The Second Art Of Darkness Right -> Stacks',
```

Comment:
Where is left -> protean and right -> protean?

Reply:
Removed for now